src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,28 @@
+package racingcar.domain;
+
+public class Car {
+
+    private static final int MOVE_CONDITION = 4;
+    private final String name;
+    private int position;
+
+    public Car(String name) {
+        this.name = name;
+        this.position = 0;
+    }
```

Comment:
정적 팩토리 메서드에 대한 내용 알아보셔도 좋을 것 같아요!
```suggestion
     private Car(String name) {
        this.name = name;
        this.position = 0;
    }
    
    public static Car create(String name) {
        return new Car(name);
    }
```

정적 팩토리 메서드는 객체 생성 로직을 클래스 내부에 숨기는 역할 즉, 캡슐화를 통해 코드를 객체 지향적으로 작성하는 데 도움을 주기 때문에 적용해 보는 것도 좋을 것 같아요!